themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -280,3 +281,15 @@ $tile-title-font-size-mobile: 3vh !default;
     }
   }
 }
+
+.no-padding-bottom {
+  .paragraph, & {
+    padding-bottom: 0 !important;
+  }
+}
+
+.no-padding-top {
+  .paragraph, & {
+    padding-top: 0 !important;
```

Comment:
1. those `!important` here seems not needed, make sure, but i think we can remove them and it should work as well - can you confirm?

2. Second thing is - do we want to clear bottom padding for every `.paragraph` that is inside `.no-padding-top/bottom`? 
i think that maybe it would be safer to add `>` ther so it would be applied only for `.paragraph` placed bellow `section.no-padding-top/bottom` 
WDYT?